packages/main/src/plugin/dashboard/dashboard-service.ts

```
@@ -43,18 +59,117 @@ export class DashboardService {
+        // Check if starting or stopping
+        if (allConnections.filter(connection => connection.status === 'starting').length) {
+          return 'Starting up...';
+        } else {
+          return 'Stopping...';
+        }
+      case HEALTH_MONITOR_STATUS.CRITICAL:
+        if (errorConnections > 1 || errorProviders > 1 || (errorConnections === 1 && errorProviders === 1)) {
+          return 'Multiple errors detected';
+        } else {
+          return 'Error detected';
+        }
```

Biome (2.4.7)

[error] This else clause can be omitted because previous branches break early.

(lint/style/noUselessElse)